frontend/src/pages/stats/index.tsx

```diff
@@ -61,10 +62,14 @@ const Stats: NextPage = () => {
           <NewContractsByDate chartStyle={chartStyle} />
         </div>
         <hr />
-        <div id="activeAccountsBydate">
+        <div id="activeAccountsByDate">
```

Comment:
@frol 
I strongly believe that one of the most disciplining practices in coding is treating PRs as [atomic entities](https://fagnerbrack.com/one-pull-request-one-concern-e84a27dfe9f1).
Can I hope to enforce that practice in explorer repo?

Reply:
@luixo Sure, I will extract this change into a separate PR.